plumpy/process_states.py

```diff
@@ -316,7 +317,7 @@ def interrupt(self, reason: Any) -> None:
     async def execute(self) -> State:  # type: ignore # pylint: disable=invalid-overridden-method
         try:
             result = await self._waiting_future
-        except Interruption:
+        except (Interruption, asyncio.CancelledError):
```

Comment:
In order to make behavior under python3.7 and python3.8 the same let's *only* re-raise for the `asyncio.CancelledError`

Reply:
done